packages/types/app/index.d.ts

```diff
@@ -15,6 +15,7 @@ type NuxtState = Record<string, any>
 export interface Context {
   app: NuxtAppOptions
   base: string
+  $config: Record<string, any>
```

Comment:
It makes sense to create a type for `Record<string, string>` so that:
a) you don't repeat yourself and risk things getting out of sync
b) so that people can externalize config options and annotate it with imported type.

Reply:
config !== env :D

Reply:
Hmm? Do you refer to me writing `Record<string, string>` instead of intended `Record<string, any>`? I've corrected myself.

Reply:
I think type of `Record<string, T>` is not fully valid because it is a recursive object with final keys of type any (or more precisely non-POJO valid types)

Reply:
https://github.com/nuxt/typescript/pull/368/files#diff-f6413043dbeb650f365479ed5cf49906R4

```ts
declare module '@nuxt/types/config/runtime' {
  interface NuxtRuntimeConfig {
    bool: boolean
  }
}
```

Reply:
![image](https://user-images.githubusercontent.com/25272043/83009472-1d085a00-a017-11ea-9fc5-61f9c61ccdc7.png)

Reply:
Why is it showing `bool`?

Reply:
Cause `$config` is of type `NuxtRuntimeConfig` which I extended